CHANGELOG.md

```diff
@@ -1,4 +1,24 @@
 # Change Log
+## [1.2.21](https://github.com/ably/ably-java/tree/v1.2.21)
+
+[Full Changelog](https://github.com/ably/ably-java/compare/v1.2.20...1.2.21)
+
+**Fixed bugs:**
+
+- Presence.endSync throws NullPointerException when processing a message [\#853](https://github.com/ably/ably-java/issues/853)
+
+**Closed issues:**
+
+- handling of channel options in InternalChannels.get is not thread safe [\#663](https://github.com/ably/ably-java/issues/663)
+- Remove hardcoded name from Maven Gradle files [\#565](https://github.com/ably/ably-java/issues/565)
+- Android CI fails due to unaccepted licenses [\#554](https://github.com/ably/ably-java/issues/554)
+- AsyncHttpScheduler.dispose\(\) is never used [\#523](https://github.com/ably/ably-java/issues/523)
+- More Encapsulation Needed [\#508](https://github.com/ably/ably-java/issues/508)
```

Comment:
I would remove all of these items as these are internal issues

Reply:
de6f2563941e1ea4983fc2aa34502f016b8c99c0